dask/array/tests/test_reductions.py

```diff
@@ -67,6 +67,8 @@ def test_reductions_1D(dtype):
     reduction_1d_test(da.nansum, a, np.nansum, x)
     with ignoring(AttributeError):
         reduction_1d_test(da.nanprod, a, np.nanprod, x)
+        reduction_1d_test(da.nancumprod, a, np.nancumprod, x)
```

Comment:
There a multiple failing tests on master so I'm not 100% sure if I've gotten all the errors, especially on older platforms because I'm running with the newest version of numpy.  I modified the code a little further to make sure these tests run via the import from the backport in the case where they are not in numpy.  So, I would say this comment on this line of code is resolved now by `256d94b`.